subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -52,6 +53,7 @@ static int download_client_callback(const struct download_client_evt *event)
 			callback(FOTA_DOWNLOAD_EVT_ERROR);
 			return err;
 		}
+		bytes_sent_to_flash += event->fragment.len;
```

Comment:
hint:
bytes_sent_to_flash == flash_img->bytes_written + flash_img->buf_bytes;
so maybe possible to not useing additional variable.

Reply:
After staring at it a bit, I decided the overhead of having to maintain one accumulator was small enough that it outweighed the potential risks of hooking into library internals that might possibly change in the future.  I left this as it was.